pkgs/by-name/lo/logiops/package.nix

```
@@ -0,0 +1,53 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  pkg-config,
+  cmake,
+  glib,
+  udev,
+  libevdev,
+  libconfig,
+}:
+
+stdenv.mkDerivation (oldAttrs: {
+  pname = "logiops";
+  version = "0.3.3";
+
+  src = fetchFromGitHub {
+    owner = "PixlOne";
+    repo = "logiops";
+    rev = "v${oldAttrs.version}";
+    sha256 = "sha256-9nFTud5szQN8jpG0e/Bkp+I9ELldfo66SdfVCUTuekg=";
+    # In v0.3.0, the `ipcgull` submodule was added as a dependency
+    # https://github.com/PixlOne/logiops/releases/tag/v0.3.0
+    fetchSubmodules = true;
+  };
+
```

```suggestion

  patches = [
    ./0001-Make-DBUS_SYSTEM_POLICY_INSTALL_DIR-externally-overr.patch
  ];

```

this is a more standard formatting we use

---

Running [`nixpkgs-fmt`](https://github.com/nix-community/nixpkgs-fmt) seems to prefer inlining it, although it also seems to prefer moving the `oldAttrs: ` function call to its own line. Would it make more sense to either a) merge it in as it is now, and worry about formatting later, or b) wait until [the Nix formatting RFC](https://discourse.nixos.org/t/rfc-0166-fcp-nix-formatting/40092) gets finalized?

---

nixpkgs-fmt shouldn't have any opinion to this in my experiences but that wasn't the point of my comment. patches is usually ordered right after src in the order it is used, not alphabetical.

---

I agree with right after `src`. But generally these formatting questions are not worth the time it takes to process them. Just commit it yourself if you'd like. Otherwise, it's okay if an attr is not in exactly the nicest order. It's a shared volunteer project.